tools/.gitignore

```diff
@@ -1,12 +1,85 @@
-/Randomizer.zip
```

Comment:
Please undo the changes to this file.
It seems you copied over the gitignore in the repository root to here.

Reply:
Hmm it seems pulling created some issues, hopefully nothing else got messed up.